docs/release-notes/rockcraft-1-7-0.rst

```diff
@@ -0,0 +1,101 @@
+.. _release-1.7.0:
+
+Rockcraft 1.7.0 release notes
+=============================
+
+20 December 2024
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.7.0.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+What's new
+----------
+
+Rockcraft 1.7.0 brings the following features, integrations, and improvements.
+
+
+Support for remote builds
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We've added support for remote builds, which are powered by Canonical's
+Launchpad platform and allow you to build rocks for multiple architectures with
+a single command.
+
+This feature is experimental and subject to change. Feel free to provide
+feedback on the `Ubuntu Discourse`_ forum.
+
+Minor features
+--------------
+
+Rockcraft 1.7.0 brings the following minor changes.
+
+
+"init" command
+~~~~~~~~~~~~~~
+
+- The base of the project file generated by the "rockcraft init" command has
+  been updated from "ubuntu@22.04" to "ubuntu@24.04".
+
+
+Rockcraft snap
+~~~~~~~~~~~~~~
+
+- The Rockcraft snap has been updated from core22 to core24.
+
+
+Go extension
+~~~~~~~~~~~~
+
+- The documentation for the "go-framework" extension has been updated with
+  improved tutorial and reference.
+
+
+OCI image configuration
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- When using "run-user: _daemon_", the value of the "Config.User" key in the
+  generated rock has been updated from the name ("_daemon_") to the numeric id.
+
+
+12-factor documentation
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- The documentation for the various 12-factor framework extensions has been
+  updated for clarity and to better conform with Canonical's guidelines.
+
+
+Maven and Ant plugins
+~~~~~~~~~~~~~~~~~~~~~
+
+- Updated the Maven and Ant plugins to generate the "/usr/bin/java" path to the
+  Java runtime executable, instead of the "/bin/java" symlink which conflicts
+  with the user-merged Ubuntu base.
+
+
+Fixed bugs and issues
+---------------------
+
+The following issues have been resolved in Rockcraft 1.7.1:
+
+- #659 Kubernetes ``runAsNonRoot`` incompatibility with ``run-user: _daemon_``.
+- #749 Trying to build the output of "expand-extensions" fails because of the
+  pebble part.
+
+
+Contributors
+------------
+
+We would like to express a big thank you to all the people who contributed to
+this release.
+
+`@dariuszd21 <https://github.com/dariuszd21>`_,
```

Comment:
In terms of visual presentation, I'm not sure I like splitting the names across lines. I'd prefer one big paragraph.

Reply:
It's rendered as a single paragraph; they are split over sequential lines here because of the line-length limit, which basically enforces "one per line" in this case
![image](https://github.com/user-attachments/assets/e17eebc4-9ed4-4e52-af33-7fc4da407e6b)

Reply:
Ah! I reacted instinctively to the code and not the preview, oops.